op5build/monitor-ninja.spec

```diff
@@ -232,6 +234,7 @@ sed -i 's/expose_php = .*/expose_php = off/g' /etc/php.ini
 %dir /var/log/op5/ninja
 
 %attr(640,%daemon_user,%daemon_group) %prefix/application/config/database.php
+%attr(644,%daemon_user,%daemon_group) %_sysconfdir/logrotate.d/ninja
```

Comment:
The logrotate configuration file should be owned by root.

Reply:
updated. Thnx